pages/linux/fdisk.md

```diff
@@ -1,6 +1,7 @@
 # fdisk
 
 > A program for managing partition tables and partitions on a hard disk.
+> See also: `partprobe`.
```

Comment:
I don't think that should be in here.

Reply:
I will rebase and remove that commit.

I don't know how that slipped in there, I normally know why it used them commits but I don't have a clue why here.

Reply:
Oh, its not a commit!?

Ok. I'll fix however I want to see if I can do it through editing a commit rather than just adding another 1 to remove it

Reply:
```suggestion
```